docs-ref-conceptual/interactive-azure-cli.md

```diff
@@ -78,19 +78,31 @@ az>>
 
 ## Query
 
-You can execute a JMESPath query on the results of the last command that you executed.
+You can execute a JMESPath query on the results of the last command that you executed by using `??`followed by a JMESPath query.
 For example, after you create a VM, you can make sure it has fully provisioned.
 
 ```azurecli
 az>> vm create --name myVM --resource-group myRG --image UbuntuLTS --no-wait -o json
-az>> ? [*].provisioningState
+az>> "?? [*].provisioningState"
 ```
 
 ```json
```

Comment:
Please remove the output blocks, since they may create an expectation of identical output.